tests/models/deta/test_modeling_deta.py

```diff
@@ -267,19 +276,19 @@ def test_config(self):
         self.config_tester.check_config_can_be_init_without_params()
 
     def test_deta_model(self):
-        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        config_and_inputs = self.model_tester.prepare_config_and_inputs(model_class_name="DetaModel")
         self.model_tester.create_and_check_deta_model(*config_and_inputs)
 
     def test_deta_freeze_backbone(self):
-        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        config_and_inputs = self.model_tester.prepare_config_and_inputs(model_class_name="DetaModel")
         self.model_tester.create_and_check_deta_freeze_backbone(*config_and_inputs)
 
     def test_deta_unfreeze_backbone(self):
-        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        config_and_inputs = self.model_tester.prepare_config_and_inputs(model_class_name="DetaModel")
         self.model_tester.create_and_check_deta_unfreeze_backbone(*config_and_inputs)
 
     def test_deta_object_detection_head_model(self):
-        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        config_and_inputs = self.model_tester.prepare_config_and_inputs(model_class_name="DetaForObjectDetection")
```

Comment:
Except for this model_tester all is DetaModel, only this model_tester is DetaForObjectDetection